modules/computer/modules.c

```diff
@@ -124,7 +124,7 @@ scan_modules_do(void)
 #define NONE_IF_NULL(var) (var) ? (var) : "N/A"
 
 	/* create the module information string */
-	strmodule = g_strdup_printf("[Module Information]\n"
+	strmodule = g_strdup_printf(_("[Module Information]\n"
```

Comment:
I wrote about this at Issue #100, with several reasons why, but I think it is better to break the strings out of the c-format string.
For example, this could be:
```
    strmodule = g_strdup_printf("[%s]\n"
                    "%s=%s\n"
                    "%s=%.2f %s\n"
                    "[%s]\n"
                    "%s=%s\n"
                    "%s=%s\n"
                    "%s=%s\n"
                    "[%s]\n"
                    "%s=%s\n"
                    "%s=%s\n",
                    _("Module Information"),
                    _("Path"), NONE_IF_NULL(filename),
                    _("Used Memory"), memory / 1024.0, _("KiB"),
                    _("Description"),
                    _("Name"), modname,
                    _("Description"), NONE_IF_NULL(description),
                    _("Version Magic"), NONE_IF_NULL(vermagic),
                    _("Copyright"),
                    _("Author"), NONE_IF_NULL(author),
                    _("License"), NONE_IF_NULL(license));
```